rfcs/2021-04-07-6330-emitting-multiple-log-events-from-remap-transform.md

```diff
@@ -0,0 +1,182 @@
+# RFC 6330 - 2021-04-07 - Emitting multiple log events from remap transform
+
+This RFC describes an approach for emitting multiple log events from a remap transform.
+
+[Credit to @JeanMertz](https://github.com/timberio/vector/pull/7038#discussion_r609476064) for this revised approach.
+
```

Comment:
```suggestion
```

If I had to give credits to everyone that helped me over the years, you'd be reading a novel for every PR or RFC I'd write 😄